mods/flora/sfcave_flora.lua

```diff
@@ -0,0 +1,533 @@
+minetest.register_node("flora:vines_horizontal", {
+	description = "Horizontal Vines",
+	tiles = {"flora_vines_horizontal.png"},
+	inventory_image = "flora_vines_horizontal.png",
+	drawtype = "firelike",
+	paramtype = "light",
+	sunlight_propagates = true,
+	walkable = false,
+	floodable = true,
+	visual_scale = 2.0,
+	drop = {
+		items = {
+			{
+				rarity = 1,
+				items = {"blocks:stick"},
+			},
+			{
+				rarity = 1,
+				items = {"flora:leaves"},
+			},
+			{
+				rarity = 2,
+				items = {"flora:leaves 2"},
+			},
+			{
+				rarity = 3,
+				items = {"blocks:stick 2"},
+			},
+			{
+				rarity = 7,
+				items = {"blocks:moonstone_powder"},
```

Comment:
why does vines drop moonstone powder as well?

Reply:
low chance, very low chance

Reply:
Is there a logic behind this? Since these typical vines don't glow as well.

Reply:
There could be a bit of powder on any vines since those glowing ones share the same surroundings. And also 1/7 is very rare

Reply:
Thats my logic at least

Reply:
Ok, marking resolved.